tcod-sys doesn't build on OSX

Hey, having some trouble building tcod on OSX.

```
anathema@Adorjan ~/C/r/ferl> cargo build
   Compiling tcod-sys v2.0.11
failed to run custom build command for `tcod-sys v2.0.11`
Process didn't exit successfully: `/Users/anathema/Code/rust/ferl/target/debug/build/tcod-sys-2a137dae9d1f93a4/build-script-build` (exit code: 101)
--- stdout
STDOUT: \rm -f /tmp/libtcod/release/bresenham_c.o /tmp/libtcod/release/bsp_c.o /tmp/libtcod/release/color_c.o /tmp/libtcod/release/console_c.o /tmp/libtcod/release/fov_c.o /tmp/libtcod/release/fov_circular_raycasting.o /tmp/libtcod/release/fov_diamond_raycasting.o /tmp/libtcod/release/fov_recursive_shadowcasting.o /tmp/libtcod/release/fov_permissive2.o /tmp/libtcod/release/fov_restrictive.o /tmp/libtcod/release/heightmap_c.o /tmp/libtcod/release/image_c.o /tmp/libtcod/release/lex_c.o /tmp/libtcod/release/list_c.o /tmp/libtcod/release/mersenne_c.o /tmp/libtcod/release/noise_c.o /tmp/libtcod/release/parser_c.o /tmp/libtcod/release/path_c.o /tmp/libtcod/release/sys_c.o /tmp/libtcod/release/sys_opengl_c.o /tmp/libtcod/release/sys_sdl_c.o /tmp/libtcod/release/sys_sdl_img_bmp.o /tmp/libtcod/release/sys_sdl_img_png.o /tmp/libtcod/release/tree_c.o /tmp/libtcod/release/txtfield_c.o /tmp/libtcod/release/wrappers.o /tmp/libtcod/release/zip_c.o /tmp/libtcod/release/namegen_c.o /tmp/libtcod/release/png/lodepng.o /tmp/libtcod/release/bresenham.o /tmp/libtcod/release/bsp.o /tmp/libtcod/release/color.o /tmp/libtcod/release/console.o /tmp/libtcod/release/fov.o /tmp/libtcod/release/heightmap.o /tmp/libtcod/release/image.o /tmp/libtcod/release/lex.o /tmp/libtcod/release/mersenne.o /tmp/libtcod/release/mouse.o /tmp/libtcod/release/noise.o /tmp/libtcod/release/parser.o /tmp/libtcod/release/path.o /tmp/libtcod/release/sys.o /tmp/libtcod/release/txtfield.o /tmp/libtcod/release/zip.o /tmp/libtcod/release/namegen.o /tmp/libtcod/release/gui/button.o /tmp/libtcod/release/gui/container.o /tmp/libtcod/release/gui/flatlist.o /tmp/libtcod/release/gui/hbox.o /tmp/libtcod/release/gui/image.o /tmp/libtcod/release/gui/label.o /tmp/libtcod/release/gui/radiobutton.o /tmp/libtcod/release/gui/slider.o /tmp/libtcod/release/gui/statusbar.o /tmp/libtcod/release/gui/textbox.o /tmp/libtcod/release/gui/togglebutton.o /tmp/libtcod/release/gui/toolbar.o /tmp/libtcod/release/gui/vbox.o /tmp/libtcod/release/gui/widget.o /tmp/libtcod/debug/bresenham_c.o /tmp/libtcod/debug/bsp_c.o /tmp/libtcod/debug/color_c.o /tmp/libtcod/debug/console_c.o /tmp/libtcod/debug/fov_c.o /tmp/libtcod/debug/fov_circular_raycasting.o /tmp/libtcod/debug/fov_diamond_raycasting.o /tmp/libtcod/debug/fov_recursive_shadowcasting.o /tmp/libtcod/debug/fov_permissive2.o /tmp/libtcod/debug/fov_restrictive.o /tmp/libtcod/debug/heightmap_c.o /tmp/libtcod/debug/image_c.o /tmp/libtcod/debug/lex_c.o /tmp/libtcod/debug/list_c.o /tmp/libtcod/debug/mersenne_c.o /tmp/libtcod/debug/noise_c.o /tmp/libtcod/debug/parser_c.o /tmp/libtcod/debug/path_c.o /tmp/libtcod/debug/sys_c.o /tmp/libtcod/debug/sys_opengl_c.o /tmp/libtcod/debug/sys_sdl_c.o /tmp/libtcod/debug/sys_sdl_img_bmp.o /tmp/libtcod/debug/sys_sdl_img_png.o /tmp/libtcod/debug/tree_c.o /tmp/libtcod/debug/txtfield_c.o /tmp/libtcod/debug/wrappers.o /tmp/libtcod/debug/zip_c.o /tmp/libtcod/debug/namegen_c.o /tmp/libtcod/debug/png/lodepng.o /tmp/libtcod/debug/bresenham.o /tmp/libtcod/debug/bsp.o /tmp/libtcod/debug/color.o /tmp/libtcod/debug/console.o /tmp/libtcod/debug/fov.o /tmp/libtcod/debug/heightmap.o /tmp/libtcod/debug/image.o /tmp/libtcod/debug/lex.o /tmp/libtcod/debug/mersenne.o /tmp/libtcod/debug/mouse.o /tmp/libtcod/debug/noise.o /tmp/libtcod/debug/parser.o /tmp/libtcod/debug/path.o /tmp/libtcod/debug/sys.o /tmp/libtcod/debug/txtfield.o /tmp/libtcod/debug/zip.o /tmp/libtcod/debug/namegen.o /tmp/libtcod/debug/gui/button.o /tmp/libtcod/debug/gui/container.o /tmp/libtcod/debug/gui/flatlist.o /tmp/libtcod/debug/gui/hbox.o /tmp/libtcod/debug/gui/image.o /tmp/libtcod/debug/gui/label.o /tmp/libtcod/debug/gui/radiobutton.o /tmp/libtcod/debug/gui/slider.o /tmp/libtcod/debug/gui/statusbar.o /tmp/libtcod/debug/gui/textbox.o /tmp/libtcod/debug/gui/togglebutton.o /tmp/libtcod/debug/gui/toolbar.o /tmp/libtcod/debug/gui/vbox.o /tmp/libtcod/debug/gui/widget.o libtcod.a libtcodxx.a libtcod.dylib libtcodxx.dylib libtcodgui.dylib libtcod_debug.dylib libtcodxx_debug.dylib libtcodgui_debug.dylib libtcodpy.pyc
Compiling debug src/bresenham_c.c
Compiling debug src/bsp_c.c
Compiling debug src/color_c.c
Compiling debug src/console_c.c
Compiling debug src/fov_c.c
Compiling debug src/fov_circular_raycasting.c
Compiling debug src/fov_diamond_raycasting.c
Compiling debug src/fov_recursive_shadowcasting.c
Compiling debug src/fov_permissive2.c
Compiling debug src/fov_restrictive.c
Compiling debug src/heightmap_c.c
Compiling debug src/image_c.c
Compiling debug src/lex_c.c
Compiling debug src/list_c.c
Compiling debug src/mersenne_c.c
Compiling debug src/noise_c.c
Compiling debug src/parser_c.c
Compiling debug src/path_c.c
Compiling debug src/sys_c.c
Compiling debug src/sys_opengl_c.c
Compiling debug src/sys_sdl_c.c
Compiling debug src/sys_sdl_img_bmp.c
Compiling debug src/sys_sdl_img_png.c
Compiling debug src/tree_c.c
Compiling debug src/txtfield_c.c
Compiling debug src/wrappers.c
Compiling debug src/zip_c.c
Compiling debug src/namegen_c.c
Compiling debug src/png/lodepng.c
Linking libtcod_debug.dylib

STDERR: In file included from src/sys_c.c:41:
include/libtcod.h:172:9: warning: 'MAX' macro redefined [-Wmacro-redefined]
#define MAX(a,b) ((a)<(b)?(b):(a))
        ^
/usr/include/sys/param.h:218:9: note: previous definition is here
#define MAX(a,b) (((a)>(b))?(a):(b))
        ^
In file included from src/sys_c.c:41:
include/libtcod.h:173:9: warning: 'MIN' macro redefined [-Wmacro-redefined]
#define MIN(a,b) ((a)>(b)?(b):(a))
        ^
/usr/include/sys/param.h:215:9: note: previous definition is here
#define MIN(a,b) (((a)<(b))?(a):(b))
        ^
src/sys_c.c:373:13: warning: 'sem_init' is deprecated [-Wdeprecated-declarations]
        if ( ret ) sem_init(ret,0,initVal);
                   ^
/usr/include/sys/semaphore.h:55:5: note: 'sem_init' has been explicitly marked deprecated here
int sem_init(sem_t *, int, unsigned int) __deprecated;
    ^
src/sys_c.c:403:3: warning: 'sem_destroy' is deprecated [-Wdeprecated-declarations]
                sem_destroy((sem_t *)sem);
                ^
/usr/include/sys/semaphore.h:53:5: note: 'sem_destroy' has been explicitly marked deprecated here
int sem_destroy(sem_t *) __deprecated;
    ^
4 warnings generated.
src/zip_c.c:87:30: warning: cast to 'void *' from smaller integer type 'int' [-Wint-to-void-pointer-cast]
                TCOD_list_push(zip->buffer,(void *)val);
                                           ^
1 warning generated.
osx/macsupport.m:17:22: warning: unused variable 'pool' [-Wunused-variable]
        NSAutoreleasePool  *pool = [[NSAutoreleasePool alloc] init];
                            ^
1 warning generated.
ld: warning: ignoring file lib/libz.a, file was built for archive which is not the architecture being linked (x86_64): lib/libz.a
Undefined symbols for architecture x86_64:
  "_gzclose", referenced from:
      _TCOD_zip_save_to_file in zip_c.o
      _TCOD_zip_load_from_file in zip_c.o
  "_gzopen", referenced from:
      _TCOD_zip_save_to_file in zip_c.o
      _TCOD_zip_load_from_file in zip_c.o
  "_gzread", referenced from:
      _TCOD_zip_load_from_file in zip_c.o
  "_gzwrite", referenced from:
      _TCOD_zip_save_to_file in zip_c.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [libtcod_debug.dylib] Error 1


--- stderr
thread '<main>' panicked at '`make` returned: exit code: 2', /Users/anathema/.cargo/registry/src/github.com-1ecc6299db9ec823/tcod-sys-2.0.11/build.rs:39
```

I cloned the tcod-rs repository and ran `cargo run --example keyboard` though and it worked. 

Cheers.

--- a/tcod_sys/Cargo.toml
+++ b/tcod_sys/Cargo.toml
@@ -8,6 +8,9 @@ repository = "https://github.com/tomassedovic/tcod-rs/tree/master/tcod-sys"
 authors = ["tomas@sedovic.cz"]
 links = "tcod"
 build = "build.rs"
+exclude = [
+    "libtcod/lib/libz.a"
+]
 
 [lib]
 name = "tcod_sys"

